src/main/java/com/greghaskins/spectrum/model/TaggingState.java

```diff
@@ -10,17 +12,17 @@
 /**
  * Represents the state of tagging for Spectrum - what it presently means.
  */
-class TaggingState {
+public class TaggingState {
```

Comment:
If this is getting promoted to the public API, then we might need to consider it's name. At first, I thought `TaggingState` captured the tags for a spec, not the filter of which tags to run and which to ignore. Perhaps `FilterCriteria` or something like that would be more appropriate.